### PR TITLE
[KPH3-6562] -  Added card view on the Admitted bodies Tab

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
see:- https://thepalladiumgroup.atlassian.net/browse/KHP3-6562

## Screenshots
cardview

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Review by donaldkibet — APPROVED** [review]

**Comment by donaldkibet** [comment]
@its-kios09 please resolve the multiple conflicts

**Comment by its-kios09** [comment]
@donaldkibet i have resolved the conflicts

File: packages/esm-morgue-app/src/card/compartment.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@carbon/colors';
+@use '~@openmrs/esm-styleguide/src/vars' as *;
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/card/compartment.scss:4]:
```suggestion
@use '@openmrs/esm-styleguide/src/vars' as *;
```

File: packages/esm-morgue-app/src/card/empty-compartment.component.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <div className={styles.cardView}>
+      <div className={`${styles.cardRow} ${styles.centeredContent}`}>
+        <div className={styles.cardLabelWrapper}>
+          <div className={styles.cardLabel}>1</div>
+        </div>
+        <span className={styles.noCompartment}>{t('empty', 'Empty')}</span>
+      </div>
+      <Button className={styles.assignButton} kind="primary" renderIcon={Movement}>
+        Assign
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/card/empty-compartment.component.tsx:18]: Should we translate this string

File: packages/esm-morgue-app/src/header/admitted-queue-header.scss [hunk 3]
```diff
@@ -1,33 +1,34 @@
-@use '@carbon/styles/scss/spacing';
-@use '@carbon/styles/scss/type';
-@import '~@openmrs/esm-styleguide/src/vars';
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@carbon/colors';
+@use '~@openmrs/esm-styleguide/src/vars' as *;
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/header/admitted-queue-header.scss:4]:
```suggestion
@use '@openmrs/esm-styleguide/src/vars' as *;
```

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**